strings/junkmail.txt

```diff
@@ -13,3 +13,27 @@ MONTHPEOPLE ARE REAL, AND NO I DON'T MEAN "MOTH" PEOPLE! CALENDAR CRYPTIDS! THE
 WE NEED YOUR BLOOD! WE ARE AN ANARCHO-COMMUNIST VAMPIRE COMMUNE. BLOOD ONLY LASTS 42 DAYS BEFORE IT GOES BAD! WE DO NOT HAVE NANOTRASEN STASIS! PLEASE, SEND BLOOD! THANK YOU! OR WE KILL YOU!
+TFW my paranoia-laden round-based roleplaying game set against the backdrop of a nonsensical, metal death trap masquerading as a space station is not paranoia-laden OR set against the backdrop of a nonsensical, metal death trap masquerading as a space station
+"skill issue"? Are you fucking kidding me? I spent a decent portion of my life writing all of that and your response to me is "skill issue"? Are you so mentally handicapped that the only thing you can comprehend is "skill issue" - or are you just some fucking asshole who thinks that with such a short response, he can make a statement about how meaningless what was written was? Well, I'll have you know that what I wrote was NOT meaningless, in fact, I even had my written work proof-read by several professors of literature. Don't believe me? I doubt you would, and your response to this will probably be "skill issue" once again. Do I give a fuck? No, does it look like I give even the slightest fuck about one fucking emoji? I bet you took the time to type that emoji too, I bet you sat there and chuckled to yourself for 20 hearty seconds before sending that into the mail. You're so fucking pathetic. I'm honestly considering directing you to a psychiatrist, but I'm simply far too nice to do something like that. You, however, will go out of your way to make a fool out of someone by responding to a well-thought-out, intelligent, or humorous statement that probably took longer to write than you can last in bed with a chimpanzee. What do I have to say to you? Absolutely nothing. I couldn't be bothered to respond to such a worthless attempt at a response. Do you want "skill issue" on your gravestone?
+Damn, I honestly don't want to deny this any longer. I want to date a syndicate girl. A normal girlfriend is nice and all, but does she know how to arm a nuke? Does she know all the latest equipment in the uplink? Didn't think so. I want my girl to know better than to call office bugs "suspicious devices". She doesn't give a fuck about syndicate revolvers or whatnot. She doesn't use syndicate bombs, cuz they do not give any advantage. Only clothings she would wear is chameleon. She knows she looks hot as fuck without an ID equipped. I know that too. And all that training in the syndicate isn't for nothing. Her CQC is strong as FUCK. If I were jumped by two assistants with spears, she'd shoot the spear-wielding maniacs, and whisper: "Alpha, Alpha, Delta." into my ear, as she holds me tight to assure me we aren't under fire anymore. Not to mention that we both love traitor gear. Our evenings would consist of heated arguments like whether The Sleeping Carp or the Double-Bladed Energy Sword rules the Syndicate world. All in all, I really fucking want to date and marry a syndicate girl.
```

Comment:
```suggestion
Damn, I honestly don't want to deny this any longer. I want to date a syndicate girl. A normal girlfriend is nice and all, but does she know how to arm a nuke? Does she know all the latest equipment in the uplink? Didn't think so. I want my girl to know better than to call office bugs "suspicious devices". She doesn't give a fuck about syndicate revolvers or whatnot. She doesn't use syndicate bombs, cuz they do not give any advantage. Only clothings she would wear is chameleon. She knows she looks hot as fuck without an ID equipped. I know that too. And all that training in the syndicate isn't for nothing. Her CQC is strong as FUCK. If I were jumped by two assistants with spears, she'd shoot the spear-wielding maniacs, and whisper: "Lima, Alpha, Delta." into my ear, as she holds me tight to assure me we aren't under fire anymore. Not to mention that we both love traitor gear. Our evenings would consist of heated arguments like whether The Sleeping Carp or the Double-Bladed Energy Sword rules the Syndicate world. All in all, I really fucking want to date and marry a syndicate girl.
```

I'm assuming you're trying to reference the sound byte that plays when you get traitor? It's Whiskey Echo Whiskey, Lima Alpha Delta (WEW LAD)